test/patterns/method_siblings/test_method_siblings.py

```diff
@@ -0,0 +1,20 @@
+import os
+import unittest
+from aibolit.patterns.method_siblings.method_siblings import MethodSiblings
+from pathlib import Path
+
+
+class TestMethodSiblingsSiblings(unittest.TestCase):
+    dir_path = Path(os.path.realpath(__file__)).parent
+
+    @unittest.skip("Not implemented")
+    def test_find_simple_method_siblings(self):
+        self.assertEqual(
+            [(2, 5)], MethodSiblings(Path(self.dir_path, 'SimpleMethodSiblings.java')).value()
```

Comment:
For all our patterns `value` method return type is `List[LineNumber]`. I suggest to comply with this interface. I suggest returning the lowest line number in the tuple. In that particular case it will be `2`;